macros/responseStatus.njk

```diff
@@ -42,11 +42,14 @@
   {% elseif claim.status === ClaimStatus.FREE_MEDIATION %}
     {{ t('{{ defendantName }} has rejected the claim. They’ve suggested free mediation on {{ respondedAt }}. You can accept or reject',
     { defendantName: claim.response.defendant.name, respondedAt: claim.respondedAt | date }) }}
+    <p><a href="/case/{{ claim.externalId }}/response/view-response">{{ t('View full response') }}</a></p>
```

Comment:
should use the paths variable for this and call `evaluateUri({ externalId: claim.externalId })` on it